documents/operating.md

```diff
@@ -52,14 +52,202 @@ SECTION 4: 💵💵💵 MONEY 💵💵💵
+
+A vote can be changed at any time until the decision has been determined to be final.
+
+Votes during plotting sessions override votes shown on Pull Requests, even if the same
+Corporate Overlord voted differently in both places.
+
+
+#### GitHub -> Plotting Session
+Some decisions are explored on GitHub but ultimately culminate in a live vote during
+a Plotting Session.  The process is the same as described in GitHub, but votes for
+open tickets are either scheduled into plotting sessions in advance or raised via
+a motion during the New Business portion of an agenda.
+
+
+### Voting Decisions
+The following decisions will require a formal input from the Hive Mind.
+
+
+- **Adding projects:** GitHub -> Plotting Session, 30% vote for preliminary approval,
+70% vote for final approval.  Applications are submitted to `/projects` using the
+`/documents/project.md` template.  A project representative should attempt to attend
+the relevant Plotting Session to answer questions.
+
+- **Updating projects:** GitHub -> Plotting Session, 30% vote for preliminary approval,
+70% vote for final approval.  The relevant project file should be updated via Pull Request
+in `/projects`.
+
+- **Removing projects:** GitHub -> Plotting Session, 30% vote for preliminary approval,
+70% vote for final approval.  Removing a project from BIFFUD involves a Pull Request to place
+its `.md` into `/projects/archive` and to update the file appropriately.
+
+- **Adding Corporate Overlords:** Plotting Session, 70% vote.
+- **Removing Corporate Overlords:** Plotting Session, 70% vote.
+- **Changing section 1 of `operating.md` OR changing this bullet:** GitHub -> Plotting Session, 100% vote.
+- **Changing any other section of `operating.md`:** GitHub -> Plotting Session, 70% vote.
+- **Approving expenses:** GitHub -> Plotting Session, 50% preliminary approval, 70% final approval.
+Applications are submitted to `/expenses` using the `/documents/expense.md` template.
+- **Canceling tax reimbursement:** Plotting Session, 70% vote.
+- **Creating one-time capital contributions from the Hive Mind:** Plotting Session, 100% vote.
+- **Creating one-time capital dispersements to the Hive Mind:** Plotting Session, 70% vote.
+- **Electing Corporate Overlords to named positions:** GitHub -> Plotting Session, 10% preliminary,
+Democratic Vote. Election applications are submitted to `/elections` using the `/documents/election.md`
+template. If a candidate gets 10% approval their application is merged.  Candidates must be locked in
+a full day before the election is held.
+- **Making modifications to `CONTRIBUTING.md`:** GitHub -> Plotting Session, 50% vote.
+- **Making modifications to `/corporate`:** GitHub, vote requirements
+based on change type, determined by `CONTRIBUTING.md`.
+- **Approving minutes:** GitHub, 50% vote. Pull Request made to `/meetings` using
+the `documents/meeting.md` template.
+- **Posting to official social media channels with `- HM` or `- CO`: Slack, 30%
+
```

Comment:
A couple of things need to be under a scheduled session (or with 10 days advanced notice) and _possibly_ can't be under a sudden rolling vote; ([here](http://www.legis.state.pa.us/cfdocs/legis/LI/consCheck.cfm?txtType=HTM&ttl=15&div=0&chpt=17) § 1704 (b)). (mentions a few things, but a few important ones: [ch3](http://www.legis.state.pa.us/cfdocs/legis/LI/consCheck.cfm?txtType=HTM&ttl=15&div=0&chpt=3) BIFFUD buying stake in another entity, mergers, etc; [ch19](http://www.legis.state.pa.us/cfdocs/legis/LI/consCheck.cfm?txtType=HTM&ttl=15&div=0&chpt=19) a bunch of stuff, but mostly just subchapter b § 1911: anything amending our "articles", like name changes, term of existence, business purpose, any change in shareholder rights. Possibly amending this operating agreement, once it's in effect?)

so plz specifically mark that issue as something to have considered by, and maybe written/rewritten by a lawyer